### PR TITLE
fix command line options for windows bat file

### LINKED ISSUE #15284: command line parameters like --help and -p pidfile broken on windows
The bat script is broken on windows. The parameters `!newparams!` should be placed at the ned of the line in bat script here: https://github.com/elastic/elasticsearch/blob/master/distribution/src/main/resources/bin/elasticsearch.bat#L46 as @spinscale pointed out.

We need a test for that too. This was not caught because currently we only test for the `-Des.pidfile` parameter: https://github.com/elastic/elasticsearch/blob/2.x/dev-tools/src/main/resources/ant/integration-tests.xml#L169 (have not looked at master yet). 

I can do that just want to open the issue in case someone else is dying to do that...

### LINKED ISSUE #16086: Elastic search not recognizing --cluster.name and --node.name parameter
Hi Team,

I had downloaded elasticsearch-2.1.1.zip and tried to change the node name through command line as given in the following documentation page:
https://www.elastic.co/guide/en/elasticsearch/reference/current/_installation.html

>elasticsearch.bat --cluster.name my_cluster_name --node.name my_node_name

I get the following error:
Unrecognized option: --cluster.name

### PR DESCRIPTION
**Pull request by brwe**
Options should be passed after `start`.

Closes #15284 
Closes #16086

Tests pass on windows and linux but I don't know much `ant` or `bat` so any help with improving the test is more than welcome.

### CODE REVIEW

**Comment by brwe** [comment]
I will make a separate pr for master once I figured out gradle.

**Comment by clintongormley** [comment]
@Mpdreamz could you review this please? would be good to get it in

File: qa/smoke-test-command-line-params/pom.xml [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.elasticsearch.qa</groupId>
+    <artifactId>elasticsearch-qa</artifactId>
+    <version>2.0.2-SNAPSHOT</version>
```
Comment (Mpdreamz) [inline qa/smoke-test-command-line-params/pom.xml]: had to change this to 2.0.3 to build a local distribution, not sure if this needs changing though.

**Comment by Mpdreamz** [comment]
LGTM :+1: 

no more `-Des.` ty @brwe !

**Pull request merged**